app/models/concerns/scripts/validation.rb

```diff
@@ -12,6 +12,7 @@ module Scripts::Validation
     validate :change_on_text_changed?
     validate :text_or_file_present?
     validate :no_text_and_file?
+    validate :compatible_version?, if: -> { imported_version.present? }
```

Comment:
_Creo_ que si lo ponés así debería funcionar igual:
```suggestion
    validate :compatible_version?, if: :imported_version
```

Reply:
perfecto franco. Esos truquitos jajajajaja